testdata/remote-cwl/tool1.cwl

```diff
@@ -2,7 +2,7 @@
 # We have this tool to test both local and remote packing
 
 class: CommandLineTool
-cwlVersion: v1.0
+cwlVersion: v1.2
```

Comment:
Better to make a copy with the newer version so we don't lose the test for the older version

Reply:
But the test for the older version is broken, because the `name` field should not be supported for `ArraySchema` objects in CWL v1.0.